basicsr/losses/losses.py

```diff
@@ -354,6 +354,23 @@ def forward(self, input, target_is_real, is_disc=False):
         # loss_weight is always 1.0 for discriminators
         return loss if is_disc else loss * self.loss_weight
 
+    def forward(self, input, target_is_real, is_disc=False):
+        """
+            20210525 [Lotayou]: Add support for multiscale discriminator loss
```

Comment:
It is better to describe the facts/functions, rather than the changes for the class/function docstring